PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/ui/SurveyPopUpDialog.java

```diff
@@ -0,0 +1,224 @@
+    private static final String LABEL_DO_NOT_SHOW_AGAIN = "Don't show again.";
+    private static final String LABEL_PROMPT = "<html>" +
+            "<h3 align=\"center\">Enjoy Java Toolkits?</h3>" +
+            "<p align=\"center\" style=\"line-height: 0%\"> Your feedback is important, </p>" +
+            "<p align=\"center\" style=\"line-height: 0%\"> take a minute to fill out our survey.</p>" +
+            "</html>";
+
+    private JPanel contentPane;
+    private JButton giveFeedbackButton;
+    private JButton notNowButton;
+    private HyperlinkLabel lblDoNotShowAgain;
+    private JLabel lblMessage;
+    private JLabel lblAzureIcon;
+    private JLabel lblClose;
+
+    private CustomerSurveyHelper customerSurveyHelper;
+
+    private Point dragPosition;
+    private Timer disposeTimer;
+    private LafManagerListener themeListener;
+    private Color buttonOnHoverColor = Color.WHITE;
+
+    public SurveyPopUpDialog(CustomerSurveyHelper customerSurveyHelper, Project project) {
+        super();
+
+        this.customerSurveyHelper = customerSurveyHelper;
+        this.disposeTimer = new Timer(1000* DISPOSE_TIME, (e)->this.putOff());
+        this.themeListener = lafManager -> renderUiByTheme(lafManager.getCurrentLookAndFeel());
+
+        this.setAlwaysOnTop(true);
+        this.setSize(250, 250);
+        this.setContentPane(contentPane);
+        this.setUndecorated(true);
+        this.getRootPane().setBorder(BorderFactory.createLineBorder(Color.GRAY));
+        this.setModal(false);
+        this.setLocationRelativeToIDE(project);
+        this.setDisposeTimer();
+
+        lblMessage.setText(LABEL_PROMPT);
+        lblAzureIcon.setIcon(getAzureIcon());
+
+        giveFeedbackButton.addActionListener((e) -> takeSurvey());
+        giveFeedbackButton.setFocusable(false);
+        setButtonHoverListener(giveFeedbackButton);
+
+        notNowButton.addActionListener((e) -> putOff());
+        notNowButton.setFocusable(false);
+        setButtonHoverListener(notNowButton);
+
+        lblClose.addMouseListener(new MouseInputAdapter() {
+            @Override
+            public void mouseClicked(MouseEvent e) {
+                putOff();
+            }
+        });
+
+        addDragListener();
+        // call onCancel() when cross is clicked
+        addWindowListener(new WindowAdapter() {
+            public void windowClosing(WindowEvent e) {
+                onCancel();
+            }
+        });
+        // call onCancel() on ESCAPE
+        contentPane.registerKeyboardAction(e -> onCancel(), KeyStroke.getKeyStroke(KeyEvent.VK_ESCAPE, 0), JComponent.WHEN_ANCESTOR_OF_FOCUSED_COMPONENT);
+
+        // Add listener to intellij theme change
+        LafManager.getInstance().addLafManagerListener(this.themeListener);
+        renderUiByTheme(LafManager.getInstance().getCurrentLookAndFeel());
+    }
+
+    private void renderUiByTheme(UIManager.LookAndFeelInfo theme){
+        switch (theme.getName().toLowerCase()){
```

Comment:
How to determine the dark theme by `UIUtils.isUnderDarcula()`?

Reply:
Nice idea, fixed.